su-exec.c

```diff
@@ -29,7 +29,7 @@ int main(int argc, char *argv[])
 
 	argv0 = argv[0];
 	if (argc < 3)
-		usage(0);
+		usage(1);
```

Comment:
According to `/usr/include/sysexits.h` (Linux) / `man 3 sysexits` (*BSD), shouldn't we return `64` (`EX_USAGE`) here, to comply with POSIX standard?

> EX_USAGE -- The command was used incorrectly, e.g., with the wrong number of arguments, a bad flag, a bad syntax in a parameter, or whatever.

Reply:
@michaelbaudino thanks for the review. I'll update the PR